src/core/chunked_stream.js

```diff
@@ -27,17 +27,27 @@
   }
 }(this, function (exports, sharedUtil) {
 
-var MissingDataException = sharedUtil.MissingDataException;
-var arrayByteLength = sharedUtil.arrayByteLength;
-var arraysToBytes = sharedUtil.arraysToBytes;
-var assert = sharedUtil.assert;
-var createPromiseCapability = sharedUtil.createPromiseCapability;
-var isInt = sharedUtil.isInt;
-var isEmptyObj = sharedUtil.isEmptyObj;
-
-var ChunkedStream = (function ChunkedStreamClosure() {
-  function ChunkedStream(length, chunkSize, manager) {
-    this.bytes = new Uint8Array(length);
+  var MissingDataException = sharedUtil.MissingDataException;
```

Comment:
Why is the indenting changed for these variables? Let's keep it the way it was as that is how it's done in all other files and reduces the overall size of the diff.